Hello Programs/hello.py

```diff
@@ -1 +1,5 @@
-print ("Chocolate")
+<<<<<<< HEAD
+print ("Hello")
+=======
+print ("Hello")
+>>>>>>> fabc02c1827f0a4d53d524e9874639f601e86a9f
```

Comment:
So I see you solve the conflict but now you see these weird lines of gibberish coming in that wasn't there before. If you check out the `Files Changed` tab on the top, and navigate to the changes of this file. You will see earlier the file was `print ("Chocolate")` and was just 1 line. You have 5 lines now. Why is this happening you may think.

It's safe to safe it isn't gibberish and it really means something. This is git's way of telling you that there are basically 2 versions of the file. And you have to choose which version you want to keep. I would really ask you to read through this tutorial given below line by line especially point 5 and 6. 

https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line

If you still don't get it, we will talk more about this. Have fun

Reply:
@sritanmay001 How's it going? Got it resolved yet.